gpt4all-chat/chatlistmodel.h

```diff
@@ -194,6 +194,7 @@ class ChatListModel : public QAbstractListModel
 
     Chat *currentChat() const
     {
+        Q_ASSERT(m_currentChat);
```

Comment:
Have you tried with a debug build? This assertion immediately fails for me, even when starting with a clean config.